tests/performance/common/common.go

```diff
@@ -17,16 +17,22 @@ limitations under the License.
 package common
 
 import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"io/ioutil"
 	"net/http"
 	"os/exec"
+	"strconv"
 	"strings"
 	"time"
 
+	"github.com/kubeedge/beehive/pkg/common/log"
 	"github.com/kubeedge/kubeedge/tests/e2e/utils"
-
+	"github.com/kubeedge/kubeedge/tests/stubs/common/constants"
```

Comment:
please seperate kubeedge and vendor packages.
Applicable to similar places

Reply:
Thanks. I will